egov-dss-dashboards/dashboard-analytics/ChartApiConfig.json

```diff
@@ -156,5 +156,110 @@
+      }
+    ],
+    "chartType": "metric",
+    "valueType": "number",
+    "drillChart": "none",
+    "documentType": "_doc",
+    "action": "",
+    "aggregationPaths": [
+      "Total Population Covered"
+    ],
+    "insight": { },
+    "_comment": " "
+  },
+  "provincialOrDistrictPopulationTargeted": {
+    "chartName": "DSS_HEALTH_PROVINCIAL_DISTRICT_POPULATION_TARGETED",
+    "queries": [
+      {
+        "module": "COMMON",
+        "dateRefField": "",
+        "requestQueryMap": "{\"province\": \"province\", \"district\": \"district\"}",
+        "indexName": "micro-plan-index-v1",
+        "aggrQuery": "{\n  \"aggs\": {\n    \"AGGR\": {\n      \"filters\": {\n        \"filters\": {\n          \"Province Filter\":{\n            \"bool\": {\n              \n            }\n          }\n        }\n        },\n      \"aggs\": {\n        \"Target per day\": {\n          \"sum\": {\n            \"field\": \"targetPerDay\"\n          }\n        },\n        \"Overall Target\": {\n          \"sum\": {\n            \"field\": \"overallTarget\"\n          }\n        },\n        \"Campaign Total Duartion in days\": {\n          \"sum\": {\n            \"field\": \"campaignDurationInDays\"\n          }\n        },\n        \"Target Population per Day\": {\n          \"bucket_script\": {\n            \"buckets_path\": {\n              \"tpd\": \"Target per day\"\n            },\n            \"script\": \"(params.tpd) * 1.8 * 365\"\n          }\n        },\n        \"Overall Population Target\": {\n          \"bucket_script\": {\n            \"buckets_path\": {\n              \"opt\": \"Overall Target\"\n            },\n            \"script\": \"(params.opt) * 1.8\"\n          }\n        }\n      }\n      }\n    }\n  }"
+      }
+    ],
+    "chartType": "metric",
+    "valueType": "number",
+    "drillChart": "none",
+    "documentType": "_doc",
+    "action": "",
+    "aggregationPaths": [
+      "Target Population per Day"
+    ],
+    "postAggregationTheory" : "repsonseToDifferenceOfDates",
+    "insight": { },
+    "_comment": " "
+  },
+  "provincialOrDistrictPopulationCoverageAchieved": {
+    "chartName": "`DSS_HEALTH_PROVINCIAL_DISTRICT_POPULATION_COVERAGE_ACHIEVED`",
```

Comment:
You can change this to "DSS_HEALTH_OVERVIEW_..." instead of having PROVINCIAL_DISTRICT 
@neel-tw thoughts?